### PR TITLE
Genesis GV70 2023: add missing FW versions

### PR DESCRIPTION
**Pull request by Luckst4r**
Fingerprint GV70

### CODE REVIEW

**Comment by Luckst4r** [comment]
[Cabana here](https://cabana.comma.ai/?route=5d327dec6ec07b6e%7C2023-01-04--12-07-39&exp=1704393867&sig=NW5YUox5K78usVHonEuUaaCzMXk8SVqQ3gIXvWuGnFE%3D&max=7&url=https%3A%2F%2Fchffrprivate.azureedge.net%2Fchffrprivate3%2Fv2%2F5d327dec6ec07b6e%2F9b6ec7de200ad0f44c850f67c35b3fc8_2023-01-04--12-07-39)

File: selfdrive/car/hyundai/values.py [hunk 1]
```diff
@@ -1517,9 +1517,11 @@ class Buttons:
   CAR.GENESIS_GV70_1ST_GEN: {
     (Ecu.fwdCamera, 0x7c4, None): [
       b'\xf1\x00JK1 MFC  AT USA LHD 1.00 1.04 99211-AR000 210204',
+      b'\xf1\x00JK1 MFC  AT USA LHD 1.00 1.01 99211-AR200 220125',
     ],
     (Ecu.fwdRadar, 0x7d0, None): [
       b'\xf1\x00JK1_ SCC FHCUP      1.00 1.02 99110-AR000         ',
+      b'\xf1\x8799110AR200\xf1\x00JK1_ SCC FHCUP      1.00 1.00 99110-AR200         ',
```
Comment (sshane) [inline selfdrive/car/hyundai/values.py:1524]:
```suggestion
      b'\xf1\x00JK1_ SCC FHCUP      1.00 1.00 99110-AR200         ',
```

**Comment by Luckst4r** [comment]
If you need another test drive file lmk. Happy to do another one that is longer perhaps

**Comment by sshane** [comment]
Looks good!

**Pull request merged**